gensrc/thrift/DataSinks.thrift

```diff
@@ -189,6 +189,7 @@ struct TDataStreamSink {
   11: optional i64 tablet_sink_txn_id
   12: optional Types.TTupleId tablet_sink_tuple_id
   13: optional list<Exprs.TExpr> tablet_sink_exprs
+  14: optional bool is_merge
```

Comment:
does `ordered` is a better name?

Reply:
The intuition behind this is sink node must receive reply message from exchange before starting a new data block transfer. "order" is one of the most common senario, but not all.